app/src/main/java/org/sagebionetworks/research/mpower/profile/ProfileFragment.java

```diff
@@ -48,4 +61,16 @@ public void onDestroyView() {
         super.onDestroyView();
         unbinder.unbind();
     }
+
+    @OnClick(R.id.licenses_button)
+    public void onLicensesClicked() {
+        LOGGER.debug("Clicked on the licenses button");
+        String path =  MpResourceManager.getInstance().getLicense().getAbsolutePath();
+        android.content.Intent intent = org.researchstack.backbone.ui.ViewWebDocumentActivity.newIntentForPath(this.getActivity(),
+                getString(org.researchstack.backbone.R.string.rsb_settings_privacy_policy), path);

```

Comment:
Should the title really be "Privacy Policy"?

Reply:
I guess on iOS there is no title, so I'll change this to blank